tests/registry.test.ts

```diff
@@ -99,14 +97,48 @@ describe("Registry", function() {
 				}
 			}
 		}
-		let result = await Wasmer.createPackage(manifest);
-		console.log("\n== Creating a package == ");
-		console.log(result.commands)
+		await Wasmer.createPackage(manifest);
 	})
 
 
 
-	it("can publish packages", async () => {
+	it("can publish unnamed packages", async () => {
+		let manifest =
+		{
+			"package": {
+				"name": owner + "/"
+			},
```

Comment:
Unnamed packages should not have name at all

Reply:
Even though the field is named "name" in the manifest specification, what we consider the *name* of the package here is empty. `owner + "/"` only specifies the namespace.